DatabaseDesign/Database_design_SQL

```diff
@@ -0,0 +1,80 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+CREATE SCHEMA IF NOT EXISTS `mydb` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+USE `mydb` ;
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Salary`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Salary` (
+  `employee_id` INT NOT NULL,
```

Comment:
Just wondering why you don't have primary key on this table while many other tables has primary key?